packages/utils/src/ColorUtils.test.ts

```diff
@@ -0,0 +1,15 @@
+import ColorUtils from './ColorUtils';
+
+describe('isDark', () => {
+  it('returns true if the background is dark', () => {
```

Comment:
You should throw some more tests in here for format of the color string
`#000`
`black`
`rgb(0,0,0)`
`rgba(0,0,0,1)`
`hsl(0, 0%, 0%)`

Reply:
For some reason, `black` doesn't work. Should I omit that test or fix ColorUtils?

Reply:
You sure? it works for me?

Reply:
![image](https://user-images.githubusercontent.com/61912640/219698817-dee89720-e546-4149-905d-c14a2580df18.png)
It's throwing for me. Did I enter something wrong? I am calling `ColorUtils.isDark('black')`.